detect/detect.go

```diff
@@ -149,7 +149,13 @@ func (d *Detector) AddGitleaksIgnore(gitleaksIgnorePath string) error {
 	scanner := bufio.NewScanner(file)
 
 	for scanner.Scan() {
-		d.gitleaksIgnore[scanner.Text()] = true
+		line := scanner.Text()
+		// Trim leading and trailing whitespace
+		line = strings.TrimSpace(line)
```

Comment:
This could be a single statement.
```suggestion
		line := strings.TrimSpace(scanner.Text())
```